docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -199,52 +206,102 @@ Whereas in production, it would return:
 <img src="https://bigcommerce.customcdn.net/img/image.jpg" />
 ```
 
-As highlighted above, the helper is configured to rewrite *local* URLs to a `assets/cdn/` subfolder. The `stencil bundle` command will exclude this local `assets/cdn/` subfolder from the bundle that it creates. This filtering circumvents the 50 MB size limit on the resulting .zip file.
+As highlighted above, the helper is configured to rewrite *local* URLs to an `assets/cdn/` subfolder. The `stencil bundle` command will exclude this local `assets/cdn/` subfolder from the bundle that it creates. This filtering circumvents the 50 MB size limit on the resulting ZIP file.
 
 ### {{money}}
 
-Format number length, thousands delimiter, and decimal delimiter.
+```handlebars
+{{money value n s c}}
+```
+Formats number length, thousands delimiter, and decimal delimiter.
 
-[See it in GitHub](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/money.js).
+#### Parameters
+
+- `value` {Number}: The number to format.
+- `n` {Integer}: Length of decimal.
+- `s` {Mixed}: Thousands delimiter.
+- `c` {Mixed}: Decimal delimiter.
+
+#### Example
 
-[See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=money).
+```handlebars
+{{money 1000 2 ',' '.'}}
+```
+
+[See it in GitHub](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/money.js).
 
 ### {{getFontLoaderConfig}}

```

Comment:
@junedkazi Can you please provide parameters and an example for this helper?

Reply:
**Input:** https://github.com/bigcommerce/paper-handlebars/blob/ce4c27f567fb7835c1205ab54bd29913bcdc4950/spec/helpers/getFontLoaderConfig.js#L9 (Ignore the last random one)
**Output:** https://github.com/bigcommerce/paper-handlebars/blob/ce4c27f567fb7835c1205ab54bd29913bcdc4950/spec/helpers/getFontLoaderConfig.js#L21

Reply:
@junedkazi I updated the helper based on the spec. Added `fontConfig` param and `{{getFontLoaderConfig 'Google_Karla_700'}}` example.